index.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html>
+  <head>
+    <link rel='stylesheet' href='style.css'>
+  </head>
+  <body>
+    <h1 id='title'>Free CodeCamp Survey Form</h1>
+    <p id='description'>Thank you for taking the time to help us improve the platform</p>
+    <form id='survey-form'>
+      <fieldset>
+      <label id='name-label' for='name'>Name<input id='name' type='text' placeholder='Enter your name' required></label>
+      </fieldset>
+      <fieldset>
+      <label id='email-label' for='email'>Email<input id='email' type='email' placeholder='Enter your email' required></label>
+      </fieldset>
+      <fieldset>
+      <label id='number-label' for='number'>Age<input id='number' type='number' min='10' max='99' placeholder='Age' required></label>
```

Comment:
Please add a name attribute to all the input elements. checkout this link to know the purpose of using name attributes.
https://www.geeksforgeeks.org/html-input-name-attribute/